modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/details.jsp

```diff
@@ -37,6 +37,8 @@ renderResponse.setTitle(LanguageUtil.format(request, "edit-x", objectDefinition.
 				"hasUpdateObjectDefinitionPermission", objectDefinitionsDetailsDisplayContext.hasUpdateObjectDefinitionPermission()
 			).put(
 				"isApproved", objectDefinition.isApproved()
+			).put(
+				"isChildNode", objectDefinition.isChildNode()
```

Comment:
I think it would be nice to check with Feliphe/Daniel which name they use when referencing the "child node". Double-check if this term exists or if we can send this PR with this name.

Reply:
Renamed [here](https://github.com/liferay-objects/liferay-portal/pull/2529/commits/4cc046c59cbe83f862ebff16a78d80cef4c21d0e)